docs/lcdproc-dev/language.docbook

```diff
@@ -760,14 +803,22 @@
 	      given key(s). If you reserve the key(s) in shared mode, other
 	      clients can still reserve these keys too. If you reserve the key(s)
 	      in exclusive mode no other client can reserve them again.
+	      Shared mode is the default.
+	    </para>
+	    <para>
 	      Key(s) reserved in shared mode will only be returned when a screen
 	      of the current client is active. These keys can be used for
-	      interaction with a visible screen (default).
+	      interaction with a visible screen.
 	      Key(s) reserved in exclusive mode will be returned regardless of
 	      which screen is active. They can be used to trigger a special
 	      feature or to make a screen come to foreground.
+	    </para>
+	    <para>
 	      Note that you cannot reserve a key in exclusive mode when an
-	      other client has reserved it in shared mode.
+	      other client has reserved it in shared mode. However a key
+	      can be reserved in exclusive mode and on a per screen basis
+	      with <command>key_add</command> just fine. (In this case
```

Comment:
Minor edit here to move the parentheses from "In... wins." to around "with... fine."